konrad/atmosphere.py

```diff
@@ -276,15 +276,15 @@ def get_cold_point_plev(self, pmin=10e2):
 
         return plev[np.argmin(T[plev > pmin])]
 
-    def get_triple_point_plev(self, pmin=10e2):
-        """Return the triple point pressure.
+    def get_triple_point_index(self, pmin=10e2):
```

Comment:
This way the atmosphere provides a ``get_coldpoint_plev`` and a ``get_triple_point_index`` method.
For the interface, it may be nice to have all four combinations. The ``*_plev`` methods could simply call the respective ``*_index`` one and return the corresponding pressure:

```python
def get_triple_point_plev(self):
   return self['plev'][self.get_triple_point_index()]
```

I can also commit that later, if you want 😉